tests/bsim/bluetooth/audio/src/cap_commander_test.c

```diff
@@ -352,6 +824,63 @@ static void discover_cas(size_t acceptor_cnt)
 	}
 }
 
+static void discover_ba(size_t acceptor_cnt)
+{
+	k_sem_reset(&sem_ba_discovered);
+
+	for (size_t i = 0U; i < acceptor_cnt; i++) {
+		int err;
+
+		err = bt_bap_broadcast_assistant_discover(connected_conns[i]);
+		if (err != 0) {
+			FAIL("Failed to discover BASS on the sink (err %d)\n", err);
+			return;
+		}
+	}
+
+	for (size_t i = 0U; i < acceptor_cnt; i++) {
+		const int err = k_sem_take(&sem_ba_discovered, SEM_TIMEOUT);
+
+		if (err != 0) {
+			FAIL("Failed to take sem_ba_discovered for %p: %d",
+			     (void *)connected_conns[i], err);
+		}
+	}
+}
+
+static void sync_to_broadcaster(void)
```

Comment:
```suggestion
static void pa_sync_to_broadcaster(void)
```
To avoid confusion with BIG sync